tests/random_attack.py

```diff
@@ -230,14 +236,13 @@ def replace_nas_in_column(self, frame):
     def sort_columns(self, frame):
         if frame.ncols == 0:
             return
-        ncols_sort = random.randint(1,5)
-        a = self.random_array(frame.ncols, newn=ncols_sort, positive=True)
+        ncols_sort = random.randint(1, frame.ncols)
```

Comment:
This is problematic: a frame may have 100s or even 1000s columns. When you select a random integer in the range [1, ncols], you are usually selecting a very large number of columns, which will be very slow.
The most typical use-case is to sort by a single column, occasionally by few columns, and only very rarely we want to sort by many columns.

Same for creation of key down below.

Reply:
I tested both approaches, and found that even larger number of columns is ok, but may be testing smaller number of columns is better, so let's do something like `int(random.expovariate(1.0)) + 1` here.